### ISSUE TITLE
Atlas review - Warming levels

### ISSUE THREAD

**Issue by huard** [IPCC-WG1/Atlas#16]
- [x] README: should discuss difference between `CMIP6Amon_Atlas_WarmingLevels.csv` and `CMIP6_Atlas_WarmingLevels.csv`
- [x] README: typo - maintained **by** Mathias Hauser
- [x] CMIP5_WarmingLevels_spread_45.pdf: Range from Inf to -Inf is a bit weird.
- [x] scripts/: Make sure all scripts start with a short description about what the code does.

**Comment by jesusff** [comment]
Thanks for the review. The Amon version of the warming level periods has been removed from the final repository (it was a subset of the no-Amon file). All scripts in this and other folders have been documented with a header including a short description. The figures with the warming level periods will be replaced by a single panel to be included as figure in the README in this folder. The code to produce any of the panels will be included in the repo (avoiding the Inf--Inf problem).

The typo was already corrected. To reduce clutter, I took the freedom to delete your huard-patch-1 branch, which pointed here: 866fd45 just fixing this typo.
